docs/decisions/0008-frontend-plugin-fallback.rst

```diff
@@ -0,0 +1,38 @@
+How to Handle Frontend iFrame Plugin Fallback 
+======================================================================================
+
+Status
+------
+
+Draft
+
+Context
+-------
+The Frontend Plugin Framework's iFrame class implementation will live in Frontend Platform,
+but the question of how we want to handle the case where the iFrame fails to render needs to
+handled. 
+
+Given that this is a new feature that is being made available to the frontend, we 
+can't anticipate all of the ways that the plugin will be used for a given repo, nor do we 
+have any intention to limit the scope of how it can be used. Because we don't all of the use 
+cases for the plugin, we also don't know the desired fallback method that is desired for any 
+given plugin.
+
+Decision
+--------
+For the above reasons, we will have failing iFrame plugins return an H1 tag that 
```

Comment:
I'm fully aware that this is likely not the best way to deliver an error, but in hopes of stirring enough of the pot to bring people into the conversation, I'll leave it here (see [Constructive Controversy](https://www.mindtools.com/absml9b/constructive-controversy))